### PR TITLE
Cosmetic: remove redundant word "across"

### PR DESCRIPTION
**Pull request by zhjwpku**
Though this is trivial, it can make the website more neat.

Signed-off-by: Zhao Junwang <zhjwpku@gmail.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
LGTM

**Comment by zhjwpku** [comment]
Can anybody explain why the build failed?

File: doc/architecture.rst [hunk 1]
```diff
@@ -1241,11 +1241,11 @@ Data Striping
...
 scalability. So storage systems often support `striping`_--storing sequential
-pieces of information across across multiple storage devices--to increase
-throughput and performance. The most common form of data striping comes from
-`RAID`_. The RAID type most similar to Ceph's striping is `RAID 0`_, or a
-'striped volume.' Ceph's striping offers the throughput of RAID 0 striping,
-the reliability of n-way RAID mirroring and faster recovery.
+pieces of information across multiple storage devices--to increase throughput
+and performance. The most common form of data striping comes from `RAID`_.
+The RAID type most similar to Ceph's striping is `RAID 0`_, or a 'striped
+volume.' Ceph's striping offers the throughput of RAID 0 striping, the
```
Comment (tchaikov) [inline doc/architecture.rst]: could you take this chance to s/volume.'/volume'./?

**Comment by tchaikov** [comment]
and could you revise the title of your commit message like `doc/architecture.rst: remove redundant word "across"` also?

**Comment by tchaikov** [comment]
```
error: buffer::end_of_buffer
**** failed to decode ../ceph-object-corpus/archive/0.61.4-60-g24c59be/objects/RGWBucketInfo/25f5ff183bba6da708243a0bbbe40a2e ****
error: buffer::end_of_buffer
**** failed to decode ../ceph-object-corpus/archive/0.61.4-60-g24c59be/objects/RGWBucketInfo/54f53e3e301071f48247d59272b27b53 ****
```

@zhjwpku the failure was addressed by #8165, you can rebase your change against master and repush to get a clean jenkins "make check".

**Comment by zhjwpku** [comment]
@tchaikov sure, i did it again, hope this time I got it right, close this PR.

**Pull request closed without merging**